### PR TITLE
[frontend] Open most recent file when opening assignment.

### PR DESCRIPTION
**Pull request by kcolford**
follow up on #164

### CODE REVIEW

**Comment by e45lee** [comment]
So the WebSockets returns a jQuery.Deferred object which will resolve to a value once a response is received from the backend - it does not return a value which can be directly used -- so your code as it stands will not work correctly here.

We typically wrap these Deferred objects with Angular's $q library (using $q.when - there are plenty of examples in the codebase).

You can interact with these $q Deferred objects using .then and .catch (to handle success/failure, respectively).  Take a look at https://docs.angularjs.org/api/ng/service/$q

**Comment by kcolford** [comment]
This next patch addresses that

**Comment by kcolford** [comment]
Done. Also, the code builds.

**Comment by kcolford** [comment]
Bump.

This pull request has been idle for a while, so I would like to get some feedback on whether it should be merged/inproved/closed.

**Comment by e45lee** [comment]
I'll take a look at it hopefully sometime this week.

File: src/frontend/frontend/frontend.js [hunk 1]
```diff
@@ -552,6 +552,14 @@ angular.module('frontend-app', ['seashell-websocket', 'seashell-projects', 'jque
         self.close = function () {
             $state.go('list-projects');
         };
+      self.project.resentFile()
```
Comment (kpalway) [inline src/frontend/frontend/frontend.js]: Should be recentFile()

Comment (kcolford) [inline src/frontend/frontend/frontend.js]: Fixed with recent patch.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           ‎

**Pull request merged**